chapters/index.md

```diff
@@ -43,6 +45,8 @@ Mark Gisi,
 Marshall Clow,
 Martin Michlmayr,
 Martin von Willebrand,
+Mark Atwood,
+Matija Suklje,
```

Comment:
Matija Šuklje

Sorry, I know it’s a special character, but I like it :smile_cat: 

Thank you for the mention, BTW!